pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -120,6 +119,13 @@ type ReconcileCincinnati struct {
 // Reconcile reads that state of the cluster for a Cincinnati object and makes changes based on the state read
 // and what is in the Cincinnati.Spec
 func (r *ReconcileCincinnati) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	/*    **Reconcile Pattern**
+	1. Create all the kubeResources
+	2. Make a few modifications to the kubeResources
+	3. Regenerate some of the kubeResources
+	4. Ensure all the kubeResources are correct in the Cluster
+	*/
```

Comment:
Personally, I think we should drop this comment in favor of more detailed comments where appropriate. For example, where we loop over `postAddPullSecret` and `postAddExternalCACert` would be a good place to specifically mention that we are updating our reference `resources` to add a pull secret and external CA cert if found.

Reply:
Added some more detailed comments, but I kept this section to use as an index.  Let me know what you think.